client/lib/simple-payments/utils.js

```diff
@@ -1,10 +1,31 @@
 /** @format */
 
 /**
- * Internal dependencies
+ * External dependencies
  */
+import trimEnd from 'lodash/trimEnd';
```

Comment:
I'll need to change this after https://github.com/Automattic/wp-calypso/pull/28559 is merged.